apps/vs-code-designer/src/app/commands/dotNetSDK/uninstallDotNetSDK.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { dotnet, dotNetPackageName, dotNetAptPackageName, dotNetSDKMajorVersion } from '../../../constants';
+import { ext } from '../../../extensionVariables';
+import { executeCommand } from '../../utils/funcCoreTools/cpUtils';
+
+export async function uninstallDotNetSDK(): Promise<void> {
+  ext.outputChannel.show();
+
+  switch (process.platform) {
+    case 'win32':
```

Comment:
Let's put the `win32`, `linux` and `darwin` in the constant file. I see it is used a lot in your changes